iphone/Classes/TiBluetoothPeripheralManagerProxy.m

```diff
@@ -159,7 +159,7 @@ - (void)updateValueForCharacteristicOnSubscribedCentrals:(id)args
         [result addObject:[(TiBluetoothCentralProxy *)subscribedCentral central]];
     }
     
-    [peripheralManager updateValue:[(TiBlob *)value data]
+    return [peripheralManager updateValue:[(TiBlob *)value data]
```

Comment:
Does this work for you? I would suspect you need to return an `NSNumber *` and wrap the return in `NUMBOOL()`.